wdl/transforms/draft3/src/main/scala/wdl/draft3/transforms/wdlom2wom/package.scala

```diff
@@ -1,11 +1,16 @@
 package wdl.draft3.transforms
 
 import better.files.File
+import cats.instances.either._
+
 import wdl.draft3.transforms.ast2wdlom._
+import wdl.draft3.transforms.wdlom2wom.FileElementToWomExecutable.FileElementAndInputsFile
 import wom.executable.Executable
 
 package object wdlom2wom {
   implicit val fromWorkflowDefinitionElementToWorkflowDefinition = WorkflowDefinitionElementToWomWorkflowDefinition
 
-  def womFromDraft3FileConverter(inputs: Option[String]): FromAtoB[File, Executable] = FromAtoB.viaX(draft3FileElementFromFile, FileElementToWomExecutable(inputs))
+  def womFromDraft3FileConverter(inputs: Option[String]): CheckedAtoB[File, Executable] = {
```

Comment:
I think you will want this to be `def womFromDraft3FileConverter: CheckedAtoB[(File,  Option[String]), Executable]`, i.e. moving this argument into the Kleisli.  

If you go this route you might consider  creating a case class to give it better semantics